src/test/resources/json/nothing-happened.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
should this be renamed, so as not to be confused with the case where no events have happened which we map separately as "no events occured"